packages/visual-editor/src/utils/assemblyUtils.spec.ts

```diff
@@ -172,6 +190,11 @@ describe('resolveAssembly', () => {
     const node: CompositionComponentNode = {
       type: ASSEMBLY_NODE_TYPE,
       data: {
+        assembly: {
+          componentId: 'random-node-id',
```

Comment:
I see the same ids used in the entire fixture. it would be good to actually randomize them to make sure that in future the fixture will better reflect the actual data

Reply:
I'm not sure I understand the problem here, this id is technically only used once for every test case and just appears multiple times within the data as we just reference this element in the deeper children as well. 

Where would you see a case where we should use difference ids?